hooks/build

```diff
@@ -7,10 +6,6 @@
 echo "[***] Build hook running"
 
 docker build \
-    --build-arg VERSION=$(git describe --tags --always) \
```

Comment:
Per comment above, please leave this in, we don't have versions yet, but I will publish one soon.